frontend/src/metabase/lib/dom.js

```diff
@@ -220,7 +220,15 @@ export function constrainToScreen(element, direction, padding) {
   return false;
 }
 
-const isAbsoluteUrl = url => url.startsWith("/");
+function getSitePath() {
+  return new URL(MetabaseSettings.get("site-url")).pathname.toLowerCase();
```

Comment:
I need to ensure the pathname is lower case so we could compare it incase users accidentally set it using capital letters.